op-challenger/game/monitor_test.go

```diff
@@ -49,7 +51,7 @@ func TestMonitorGames(t *testing.T) {
 					}:
 						headerNotSent = false
 					case <-ctx.Done():
-						break
+						break loop
```

Comment:
We actually just want to return here since the context is only cancelled at the end of the test.